extension/docs/KcpIntegration.md

```diff
@@ -0,0 +1,26 @@
+## KCP payment
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Contents**
+
+- [KCP](#kcp)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+**Please see [Integration Guide](IntegrationGuide.md) first before continuing with this document.**
+
+### KCP
+1. Contact Adyen to enable KCP
+1. Backend creates a payment with following criteria:
```

Comment:
I would suggest use the same format between documentations, which one is better, following criteria or following properties ?

Reply:
`following criteria` is better.